masterlist.yaml

```diff
@@ -7075,42 +7075,42 @@ plugins:
         util: 'TES4Edit v4.0.3'
       - crc: 0x11EF2688
         util: 'TES4Edit v4.0.3'
+
   ### Open Cities Patches ###
-  # Bravil Docks by Greg Simmonds - OC Patch
+  - name: '(OC|OCBruma|OCC|OCLR|OCOD|OCR|OCR|COC)(\+|\-| \+ ).*(\-| )(Wells|Patch)\.esp'
```

Comment:
IIRC, you don't have to escape dashes.

Reply:
> IIRC, you don't have to escape dashes.

Yeah, possibly, they're used for range, so maybe you'd have to only escape when using something like `[0-9]`.
I added them just in case as I'm not sure whether they're always treated as literal, unless they're inside bracket.

Reply:
Yeah, https://www.rexegg.com/regex-quickstart.html doesn't mention escaping them, so I'll take them out.